### DISCUSSION TITLE
@mui Portal Component can't be exported

### DISCUSSION THREAD

**Discussion by Jerunnon** [firecmsco/firecms#223] [Q&A]
Hi,

fist of all, I really like FireCMS, it's awesome 🙌🏻. Unfortunately I ran into an error, which I don't understand.
The error says: `ERROR in ./node_modules/@camberi/firecms/dist/index.es.js 9191:13-19
export 'Portal' (imported as 'Portal') was not found in '@mui/material' (possible exports: (all the possible exports, but Portal is not shown)`. 

I tried to import it from @mui/base/portal but I get the same error. Do you have any solutions to fix this?

**package.json:**

`{
  "name": ",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@camberi/firecms": "^1.0.0-rc.1",
    "@emotion/react": "^11.7.1",
    "@emotion/styled": "^11.6.0",
    "@mui/icons-material": "^5.3.1",
    "@mui/lab": "^5.0.0-alpha.67",
    "@mui/material": "^5.4.0",
    "@mui/styles": "^5.3.0",
    "@testing-library/jest-dom": "^5.16.2",
    "@testing-library/react": "^12.1.2",
    "@testing-library/user-event": "^13.5.0",
    "@types/jest": "^27.4.0",
    "@types/node": "^16.11.22",
    "@types/react": "^17.0.39",
    "@types/react-dom": "^17.0.11",
    "firebase": "^9.6.6",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router": "^6.2.1",
    "react-router-dom": "^6.2.1",
    "react-scripts": "5.0.0",
    "typescript": "^4.5.5",
    "web-vitals": "^2.1.4"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
`

**Comment by fgatti675** [comment]
Hi Jerunon, they introduced a breaking change in `mui` 5.4, you can just revert to version `5.3.0` while we release a fix in the next version
